### PR TITLE
Update thegeeks.yml categories

### PR DESCRIPTION
**Pull request by SirLouen**
#### Description
- Added missing categories
- Removed categories that don't exist anymore
- Renamed some categories to match 
- Ordered all the categories to make it easy to spot and review future changes

#### Issues Fixed or Closed by this PR

* Missing torrent categories won't show up in the feed

### CODE REVIEW

**Comment by garfield69** [comment]
FYI, in future, the order that makes it easier for us to check is the order that the sites torrents page has in the HTML. 
I usually dump the HTML and compare the list with the indexer side by side.

File: src/Jackett.Common/Definitions/thegeeks.yml [hunk 1]
```diff
@@ -11,87 +11,88 @@ links:
...
+    - {id: 202, cat: Other, desc: "Occultism: Mythology Folklore"}
+    - {id: 202, cat: Other, desc: "Occultism: Mythology Folklore"}
+    - {id: 203, cat: Other, desc: "Paranormal"}
+    - {id: 204, cat: Other, desc: "Parapsychology"}
+    - {id: 11, cat: Other, desc: "Political Studies"}
+    - {id: 61, cat: Other, desc: "Science: Biology"}
+    - {id: 77, cat: Other, desc: "Science: Chemistry"}
     - {id: 12, cat: Other, desc: "Science: Math/Statistics"}
+    - {id: 10, cat: Other, desc: "Science: Medicine/Health "}
+    - {id: 29, cat: Other, desc: "Science: Philosophy"}
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/thegeeks.yml:83]: remove trailing space to comply with lint
```suggestion
    - {id: 29, cat: Other, desc: "Science: Philosophy"}
```

File: src/Jackett.Common/Definitions/thegeeks.yml [hunk 2]
```diff
@@ -11,87 +11,88 @@ links:
...
     - {id: 12, cat: Other, desc: "Science: Math/Statistics"}
+    - {id: 10, cat: Other, desc: "Science: Medicine/Health "}
+    - {id: 29, cat: Other, desc: "Science: Philosophy"}   
+    - {id: 76, cat: Other, desc: "Science: Physics"}
+    - {id: 14, cat: Other, desc: "Science: Psych/Sociolgy"}
+    - {id: 215, cat: Other, desc: "Sexuality/Seductn/Body Img"}
+    - {id: 22, cat: Other, desc: "Sports/Exercise/Outdoors"}
+    - {id: 24, cat: Other, desc: "Stock Media"}
+    - {id: 33, cat: Other, desc: "Style & Fashion"}
+    - {id: 73, cat: Other, desc: "Docu : Survivalism"}
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/thegeeks.yml:90]: same here
```suggestion
    - {id: 73, cat: Other, desc: "Docu : Survivalism"}
```

**Review by garfield69 — CHANGES_REQUESTED** [review]

File: src/Jackett.Common/Definitions/thegeeks.yml [hunk 3]
```diff
@@ -11,87 +11,88 @@ links:
...
+    - {id: 23, cat: Other, desc: "Hobbies: Misc"}
+    - {id: 1, cat: Other, desc: "Languages / Linguistics"}
+    - {id: 20, cat: Other, desc: "Law & Justice"}
+    - {id: 30, cat: Other, desc: "Literature"}
+    - {id: 16, cat: Other, desc: "Magic & Illusion"}
+    - {id: 206, cat: Other, desc: "Native American Studies"}
+    - {id: 201, cat: Other, desc: "Occultism: Academic / Referenc"}
+    - {id: 210, cat: Other, desc: "Occultism: Bashar"}
+    - {id: 202, cat: Other, desc: "Occultism: Mythology Folklore"}
+    - {id: 202, cat: Other, desc: "Occultism: Mythology Folklore"}
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/thegeeks.yml:75]: remove duplicate category 202
```suggestion
```

**Comment by SirLouen** [comment]
> FYI, in future, the order that makes it easier for us to check is the order that the sites torrents page has in the HTML. I usually dump the HTML and compare the list with the indexer side by side.

Yes, I also thought that, the problem is that the HTML follows a non responsive table format, with a fixed width of 5 columns, so basically only when 5 new categories are added you will see the same spots again, meanwhile everything is mixed up making it difficult to track the same order over time.

**Comment by garfield69** [comment]
Thank you.

**Comment by garfield69** [comment]
v0.22.655

**Pull request merged**